README.md

```diff
@@ -79,5 +79,56 @@ relDif  : 1.6464292361378058e-12)
 
 Complete API docs are available at [https://docs.balancer.finance/smart-contracts/api](https://docs.balancer.finance/smart-contracts/api)
 
+---
+
+## NEAR Protocol
+
+Balancer's Solidity contracts also work on the NEAR EVM. Visit https://near.org for details on the technology. The following steps will walk through how to set up a local NEAR node and run Balancer's Truffle tests.
+
+### Setup
+
+#### Get Rust
+
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+[Rustup reference site](https://rustup.rs/)
+
+#### Set up NEAR node
+
+Clone the [nearcore repository](https://github.com/near/nearcore) with:
+
+    git clone git@github.com:near/nearcore.git
+    
+Navigate to the project root:
+
+    cd nearcore
+    
+Build (this will take a while):
+
+    cargo build -p neard --release --features protocol_feature_evm,nightly_protocol_features
```

Comment:
@bowenwang1996 do we have a compiled builds with nightly features on to install instead of compiling?

Reply:
No